ext/date/php_date.c

```diff
@@ -5934,20 +5903,47 @@ PHP_METHOD(DatePeriod, __wakeup)
 	ZEND_PARSE_PARAMETERS_NONE();
 
 	period_obj = Z_PHPPERIOD_P(object);
-
```

Comment:
Please don't just add whitespace changes.

Reply:
Sure, I'll get rid of the unnecessary changes once I rebase this PR on another batch of return fixes.